ampl-converter/src/test/java/com/powsybl/ampl/converter/AmplNetworkWriterTest.java

```diff
@@ -101,6 +111,19 @@ public void writeBattery() throws IOException {
     @Test
     public void writeThreeWindingsTransformer() throws IOException {
         Network network = ThreeWindingsTransformerNetworkFactory.createWithCurrentLimits();
+        network.getThreeWindingsTransformer("3WT").getLeg1()
+                .newPhaseTapChanger()
+                .beginStep()
+                .setRho(1)
+                .setR(0.1)
+                .setX(1.)
+                .setB(0.)
+                .setG(0.)
+                .setAlpha(0)
+                .endStep()
+                .setTapPosition(0)
+                .setLowTapPosition(0)
+                .add();
```

Comment:
Is this sufficient to test everything you change in this PR?

Reply:
no :-)